nominees/brainflow.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "BrainFlow",
+  "description": "BrainFlow is a library intended to obtain, parse and analyze EEG, EMG, ECG and other kinds of data from biosensors.",
+  "website": "https://brainflow.org/",
+  "license": [
+    {
+      "spdx": "MIT",
+      "licenseURL": "https://github.com/brainflow-dev/brainflow/blob/master/LICENSE"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 3,
+      "evidenceText": "With this library neuroscientist can obtain, parse and analyze electrical brain activity (EEG), muscle activity (EMG), heart rate (ECG) and other kinds of data from biosensors. For example, it can help develop tools and software for people with disabilities."
+    }
+  ],
+  "sectors": [],
```

Comment:
@Rolikasi feel free to add  `Health` as an entry to the sectors list. Thanks.

Reply:
Thanks for a review. Done)